pkg/v2/json/deserialize.go

```diff
@@ -133,6 +133,27 @@ Skip:
 	return string(d.data[start+1 : end-1]), nil
 }
 
+func (d *deserializeState) getPropertyDotNamesByPath(path string) []string {
```

Comment:
does this need to be in the library or can we define it in our app?

Reply:
I see no way to keep it in our app.
It comes from the lib json parser [patchService::Do()](https://github.com/zelloptt/go-scim/blob/620bb7b9b2b8c6b1f344d9468f29c4737ecb3ff6/pkg/v2/service/patch.go#L112) -> [ParseValue()](https://github.com/zelloptt/go-scim/blob/620bb7b9b2b8c6b1f344d9468f29c4737ecb3ff6/pkg/v2/service/patch.go#L237) -> [DeserializeProperty()](https://github.com/zelloptt/go-scim/blob/620bb7b9b2b8c6b1f344d9468f29c4737ecb3ff6/pkg/v2/json/deserialize.go#L42) -> [parseSingleValuedProperty](https://github.com/zelloptt/go-scim/blob/620bb7b9b2b8c6b1f344d9468f29c4737ecb3ff6/pkg/v2/json/deserialize.go#L265) -> [parseComplexProperty](https://github.com/zelloptt/go-scim/blob/620bb7b9b2b8c6b1f344d9468f29c4737ecb3ff6/pkg/v2/json/deserialize.go#L193)

Reply:
ok. can you add a method doc here to explain what it does?

Reply:
Added